src/types/transactions.ts

```diff
@@ -149,3 +148,7 @@ export type CsvExportTimerangeQueryParams = {
 export type CsvExportQueryParams = CsvExportTimerangeQueryParams & {
   addressHash: AddressHash
 }
+
+// TODO: Transfer to SDK
+export type TransactionDirection = 'in' | 'out' | 'swap'
+export type TransactionInfoType = TransactionDirection | 'move' | 'pending'
```

Comment:
Will be removed once https://github.com/alephium/js-sdk/pull/149 is merged and published

Reply:
@mvaivre Done! Ready for review :)